ring-core/test/ring/middleware/test/cookies.clj

```diff
@@ -102,8 +102,13 @@
     (is (= {"Set-Cookie" (list "a=b" "c=d")}
            (:headers resp)))))
 
+(deftest wrap-cookies-unrecognized-attrs
+  (let [response {:cookies {"a" {:value "foo" :unrecognized true}}}
+        handler  (wrap-cookies (constantly response))]
+    (handler {})))
```

Comment:
There needs to be an `is` check here to make sure the cookie is transferred even with the unrecognized attribute.